website/docs/docs-section/roadmap/v3/major-changes.md

```diff
@@ -4,28 +4,30 @@ title: Major Change from V2
 sidebar_label: Major Changes
 ---
 
-Here are some of the major changes from OpenAttestation v2 Data Model
+This article describes some major changes from the OpenAttestation V2 Data Model.
 
 ## `@context`
 
-The `@context` property is now introduced as per [W3C VC Care Data Model](https://www.w3.org/TR/vc-data-model/#contexts) to allow the issuer to map such short-form aliases to the URIs required by specific verifiable credentials and verifiable presentations.
+To help the issuer map short-form aliases to the URIs required by specific verifiable credentials and verifiable presentations, the `@context` property is introduced as per [W3C VC Care Data Model](https://www.w3.org/TR/vc-data-model/#contexts).
 
-OpenAttestation has released our extension to W3C credentials context via `https://schemata.openattestation.com/com/openattestation/1.0/OpenAttestation.v3.json`.
+OpenAttestation has released the extension to W3C credentials context via `https://schemata.openattestation.com/com/openattestation/1.0/OpenAttestation.v3.json`
```

Comment:
I would say it is indeed more of "our" extension or "an" extension of the W3C context

Reply:
Style guides for software writing often do not recommend using "we" or "our" because it is ambiguous. 
Here could we change it to" OpenAttestation has released its extension..."?